docs/sources/tempo/metrics-generator/span_metrics.md

```diff
@@ -150,6 +153,39 @@ The following intrinsic kinds are available for filtering.
 - `SPAN_KIND_PRODUCER`
 - `SPAN_KIND_CONSUMER`
 
+
+#### Filter policies
+
+Each filter policy is evaluated in order for every span to determine if the
+span should be included for metrics.  Both `include` and `exclude` policies are
+used to *reject* spans from the metrics.  With no filter polices applied, all
+spans are included.
+
+{{% admonition type="note" %}}
+An exclude match will take precedence over an include match within the *same*
+policy, but not later polices.
+{{% /admonition %}}
+
+{{% admonition type="note" %}}
+A matching exclude of one policy will be ignored if a later policy `include`
+matches, and that same policy does not also have a matching `exclude` for the
+span.
+{{% /admonition %}}
```

Comment:
Changing future tense to present tense to follow our [style guidelines](https://grafana.com/docs/writers-toolkit/write/style-guide/style-conventions/#write-in-present-tense). 

```suggestion
An exclude match takes precedence over an include match within the *same*
policy, but not later polices.
{{% /admonition %}}

{{% admonition type="note" %}}
A matching exclude of one policy is ignored if a later policy `include`
matches, and that same policy does not also have a matching `exclude` for the
span.
{{% /admonition %}}
```